media/js/firefox/new.js

```diff
@@ -177,6 +184,34 @@
             show_scene(scene, true);
         }
 
+        // Pixel to be removed on July 31st, 2015 (Bug 1168440)
+        function spring_campaign_measurement () {
+            var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+            var fxMatch = navigator.userAgent.match(/Firefox\/(\d+)/);
+            var ie10Match = navigator.userAgent.match(/MSIE 10/i);
+            var w8Match = navigator.appVersion.match(/Windows NT 6.2/);
+
+            if (fxMatch && Number(fxMatch[1]) < 32) {
+                // Can't say for sure if it is 1 or 0, due to Fx bug 887703
+                _dntStatus = (_dntStatus === 'yes') ? 'Unknown' : 'Unspecified';
```

Comment:
Could we just set `_dntStatus = 'Unspecified'` here? Regardless of the `yes` check, it ends up adding the pixel, right?